libraries/core-react/src/components/Accordion/AccordionHeader.tsx

```diff
@@ -76,10 +80,12 @@ const StyledAccordionHeader = styled.div.attrs<StyledAccordionHeaderProps>(
             },
           },
         })}
-  > svg {
-    color: ${iconToken.typography.color};
-  }
-`
+    > svg {
+      color: ${iconToken.typography.color};
```

Comment:
On side-note; This might be needed to update to include `fill` to support if anyone is using other icon sets than eds-icons. 🤔